.github/workflows/build-conda-linux.yml

```diff
@@ -32,6 +32,7 @@ jobs:
     with:
       conda-package-directory: ${{ matrix.conda-package-directory }}
       repository: ${{ matrix.repository }}
+      ref: ""
```

Comment:
You can also just not pass it at all and it should just work

Reply:
I think this arg in the reusable workflow defaults to nightly: https://github.com/pytorch/test-infra/blob/main/.github/workflows/build_conda_linux.yml#L16